drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/fw/host_rpu_sys_if.h

```diff
@@ -1050,6 +1066,191 @@ struct nrf_wifi_cmd_get_stats {
 	signed int op_mode;
 } __NRF_WIFI_PKD;
 
+/**
+ * @brief This structure represents the channel parameters to configure specific channel.
+ *
+ */
+struct nrf_wifi_cmd_set_channel {
+	/** UMAC header, @ref nrf_wifi_sys_head*/
+	struct nrf_wifi_sys_head sys_head;
+	/* Interface index  */
```

Comment:
probably check the doxygen style in this document

Reply:
Taken care.